docs/source/about/paasta_principles.rst

```diff
@@ -0,0 +1,108 @@
+|   package { 'apache':                       |                                     |
+|     ensure => '2.4.17',                     |                                     |
+|   }                                         |   $ apt-get install apache=2.4.17   |
+|   $ git commit -m "upgrade apache"          |                                     |
+|   $ git push origin HEAD                    |                                     |
++---------------------------------------------+-------------------------------------+
+
+At first glance, the imperative approach looks "simpler" and maybe "easier".
+For PaaSTA, we think that having config files that *declare* how apps are
+deployed is a superior way to define your infrastructure, in the same way that
+configuration management is superior to running commands on a server.
+Specifically:
+
+* It allows config files to be viewed and edited en-mass.
+* Storing the config files in scm tool allows for easy change tracking, rollbacks,
+  and auditing.
+* Declaring intent in config files allows the infrastructure to correct itself when
+  things get out of sync, as it can reference what things "should" be.
+* Config files allow humans and other tools easy access to see the global state of
+  things, without reaching to a database.
+
+2. Git makes for a pretty good control plane
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+PaaSTA uses git as the source of truth about what versions of a service should be
+deployed for which clusters. This has some nice benifits:
+
+* No need for a database to store the desired state of the services
+* Git allows for decent auditing for who deployed what when
+* Advanced git hosting (Gitolite, Gitlab, Github) make it easy to use similar ACLs
+  that are used for comitting code, for deploying that code as well.
+
+One downside to using git as a control plane is that it means all the PaaSTA components
+might need to access all this metadata from remote environments. PaaSTA solves this
+by generating a ``deployments.json`` for each service and using `soa-configs <../soa_configs.html>_` to
+distribute it. This allows PaaSTA to extract all the metadata out of a git repo
+once, and distribute it globally to all the PaaSTA clusters. It also helps isolate
+PaaSTA from git failures or outages.
+
+3. Services should be **owned** and monitored
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Most of the PaaSTA code base assumes that team data is available for a service.
+That is because at Yelp, all services are owned by some team.
+
+This principle manifests in PaaSTA through the ``monitoring.yaml`` file. The
+minimal amount of data required in that file is the ``team``. Additionally
+we encourage services to have at least a ``description`` and ``external_link``
+in `soa-configs <../soa-configs.html>_`.
+
+This helps emphasize that PaaSTA is built for the long-haul, and designed to
+run things in a production setting. Things are monitored by default, and alerts
```

Comment:
s/things/services/

You can keep the second "Things" since I think that supports your broader point (*everything* is monitored by default)